src/common/pages/entry.tsx

```diff
@@ -119,10 +119,11 @@ class EntryPage extends BaseComponent<Props, State> {
     }
 
     componentDidUpdate(prevProps: Readonly<Props>): void {
-        const {location} = this.props;
-        if (location.pathname !== prevProps.location.pathname) {
+        const { location } = this.props;
+        let entry = this.getEntry();
+        if (location.pathname !== prevProps.location.pathname && entry) {
             this.ensureEntry();
-            if(location.pathname.includes('re-')){
+            if(entry.parent_author){
                 let entry = this.getEntry();
```

Comment:
2nd `this.getEntry` is not necessary, right?

Reply:
Ah it is local mapping get so if ensureEntry updates it, it will get updated object, reverting removed I just did.